module-2/backend/week_one.md

```diff
@@ -5,30 +5,76 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  Get - it retrieves informatiom
+  Post - it sends the information
+  Puts - it displays the current information
+  Delete - it deletes the information
+
 2. What is Sinatra?
+  Sinatra is a "server" implemented in Ruby that is used for writing web apps.
+
 4. What is MVC?
+    MVC(models, views, controllers) is the format that is used for writing web apps.

```

Comment:
Try to be a little more specific with these. What is the responsibility of each piece?